pkg/network/ndmz/ndmz.go

```diff
@@ -59,6 +59,14 @@ func Create(nodeID pkg.Identifier) error {
 		if _, err := sysctl.Sysctl("net.ipv6.conf.all.forwarding", "1"); err != nil {
 			return errors.Wrapf(err, "failed to enable ipv6 forwarding in gateway namespace")
 		}
+		lo, err := netlink.LinkByName("lo")
```

Comment:
there is a function in ifaceutil package to do this: https://github.com/threefoldtech/zos/blob/70d05f1a5d2b0af1cff3fecb1d43d0a8502fb26a/pkg/network/ifaceutil/interface.go#L120-L131